### PR TITLE
[com_redirect] Improve disabled collect url in redirect plugin message

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

Following PR https://github.com/joomla/joomla-cms/pull/10688 that improve the link in com_redirect when the redirect plugin is disabled, this PR does similar change to the collect URL disable message.

Some other changes:
- The Collect URL warning is now a notice
- When redirect plugin was enabled AND collect URL option is enabled a notice is displayed. This notice was removed as it's not consistent with the other component list views.
#### Testing Instructions
1. USe latest staging
2. Go to Extensions -> Plugins -> Redirect (System) enable the plugin and disable collect urls
3. Go to Components -> Redirect, you will see the following message
   image
4. Apply patch 
5. Go to Components -> Redirect, you will see the following message
   image
6. Now enable the collect URL in the redirect plugin, you'll see no message

### CODE REVIEW

**Comment by wojsmol** [comment]
@andrepereiradasilva See https://joomla.github.io/coding-standards/?coding-standards/chapters/php.md elseif should be one wod

**Comment by andrepereiradasilva** [comment]
thanks, done

File: administrator/language/en-GB/en-GB.com_redirect.ini [hunk 1]
```diff
@@ -11,7 +11,7 @@ COM_REDIRECT_BUTTON_UPDATE_LINKS="Update Links"
 COM_REDIRECT_CLEAR_FAIL="Failed to delete unpublished links."
 COM_REDIRECT_CLEAR_SUCCESS="All unpublished links have been successfully deleted."
 COM_REDIRECT_COLLECT_URLS_ENABLED="The option 'Collect URLs' is enabled."
-COM_REDIRECT_COLLECT_URLS_DISABLED="The option 'Collect URLs' is disabled."
+COM_REDIRECT_COLLECT_URLS_DISABLED="The 'Collect URLs' option in the <a href="_QQ_"%s"_QQ_">Redirect System Plugin</a> is disabled. Error page URLs will not be collected by this component."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_redirect.ini]: again can you add
.... unless the plugin is enabled.

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on cc913a1fdf0247656328987c597c7e4b36786fe7<br /><br />> [05-Jun-2016 23:27:29 Europe/London] PHP Fatal error:  Call to undefined method RedirectHelper::getRedirectPluginId() in /Applications/MAMP/htdocs/joomla-cms-staging/administrator/components/com_redirect/views/links/view.html.php on line 58
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10735">issues.joomla.org/joomla-cms/10735</a>.</sub>

**Comment by brianteeman** [comment]
oops i might not be on current staging<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10735">issues.joomla.org/joomla-cms/10735</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on cc913a1fdf0247656328987c597c7e4b36786fe7<br /><br />Yeah - i wasnt on latest staging - all good<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10735">issues.joomla.org/joomla-cms/10735</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on cc913a1fdf0247656328987c597c7e4b36786fe7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10735">issues.joomla.org/joomla-cms/10735</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10735">issues.joomla.org/joomla-cms/10735</a>.</sub>

**Comment by wilsonge** [comment]
Merged - thanks :)

**Pull request merged**